storybook/storybook-react/src/Footer/Footer.stories.tsx

```diff
@@ -81,7 +81,7 @@ export const Default: Story = {
               </UnorderedList>
             </div>
           </Grid.Cell>
-          <Grid.Cell span={3} start={{ narrow: 1, medium: 1, wide: 9 }}>
+          <Grid.Cell span={{ narrow: 4, medium: 8, wide: 4 }} start={{ narrow: 1, medium: 1, wide: 9 }}>
```

Comment:
I don't think this should change, the third column should have the same dimensions as the other 2

Reply:
Alright, reverted.